converter/internal/common/convert_targets.go

```diff
@@ -19,17 +19,13 @@ var _ river.Capsule = ConvertTargets{}
 
 func (f ConvertTargets) RiverCapsule() {}
 func (f ConvertTargets) RiverTokenize() []builder.Token {
+	expr := builder.NewExpr()
 	var toks []builder.Token
 
-	targetCount := 0
-	for _, targetMap := range f.Targets {
-		targetCount += len(targetMap)
-	}
-
+	targetCount := len(f.Targets)
 	if targetCount == 0 {
-		toks = append(toks, builder.Token{Tok: token.LBRACK})
-		toks = append(toks, builder.Token{Tok: token.RBRACK})
-		return toks
+		expr.SetValue(f.Targets)
+		return expr.Tokens()
 	}
 
 	if targetCount > 1 {
```

Comment:
Is this always true?

Reply:
not if targetCount == 1 we can include with without the `concat()`